tests/misc/recursive_types.c3

```diff
@@ -4,21 +4,8 @@ typedef[T] LinkedListNode1 : {
     data: T
 }
 
-typedef[T] Wrapper : {data: T}
-typedef[T] LinkedListNode2 : {
-    next: Wrapper<LinkedListNode2<T>>&,
-    data: T
-}
-
-typedef[T] LinkedListNode3 : {
-    next: Wrapper<LinkedListNode3<T>&>,
-    data: T
-}
-
 function main : () -> int = {
     let a : LinkedListNode1<int>;
-    let b : LinkedListNode2<int>;
-    let c : LinkedListNode3<int>;
     return 0;
```

Comment:
?

Reply:
Whoops. I deleted that for debugging. I actually amended it yesterday but I forgot to push :face_in_clouds: